tools/container/crio/crio.go

```diff
@@ -222,6 +223,7 @@ func setupConfig(o *options) error {
 
 	cfg, err := crio.New(
 		crio.WithPath(o.Config),
+		crio.WithReference(toml.FromFile(o.Config)),
```

Comment:
I think `WithConfigReference` may be more suggestive of the struct field's role and purpose

Reply:
I think `WithConfigSource` is clearer. Have updated.